tests/unit/lms/services/grouping_test.py

```diff
@@ -29,42 +29,28 @@ def test_upsert_with_parents_inserts(self, svc, db_session):
 
         assert db_session.query(CanvasGroup).one() == test_grouping
 
-    def test_upsert_with_parents_updates(self, svc, db_session):
+    def test_upsert_with_parent_updates_existing_groupings(self, svc, db_session):
         course = factories.Course()
-        tool_consumer_instance_guid = (
-            course.application_instance.tool_consumer_instance_guid
-        )
-        lms_id = "lms_id"
+        kwargs = {
+            "tool_consumer_instance_guid": course.application_instance.tool_consumer_instance_guid,
+            "lms_id": "lms_id",
+            "parent": course,
+            "type_": Grouping.Type.CANVAS_GROUP,
+        }
```

Comment:
These same kwargs are used in both calls to `upsert_with_parent()`. Gathering them into a variable removes some noise and duplication and in the call to `upsert_with_parent()` the values that actually matter (`new_name` and `new_extra`) are now easier to see:

<pre><code>svc.upsert_with_parent(<b>lms_name=old_name, extra=old_extra</b>, **kwargs)</code></pre>